webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/snippet/configuration/DirectiveManagement.scala

```diff
@@ -246,24 +246,24 @@ class DirectiveManagement extends DispatchSnippet with Loggable {
               </style>
               <div class="jumbotron">
                 <h1>Directives</h1>
-                <p>A Directive is an instance of a Technique, which allows to set values for the parameters of the latter.</p>
-                <p>Each Directive can have a unique name, and should be completed with a short and a long description, and a collection of parameters for the variables defined by the Technique.</p>
+                <p>A directive is an instance of a technique, which allows to set values for the parameters of the latter.</p>
+                <p>Each directive can have a unique name, and should be completed with a short and a long description, and a collection of parameters for the variables defined by the technique.</p>
                 <p>Techniques are often available in several versions, numbered X.Y, X being the major version number and Y the minor version number:</p>
                 <ol>
                   <li><b>Bugs</b> are fixed in all existing versions of Rudder techniques. Make sure you update your Rudder packages frequently.</li>
                   <li>A new <b>minor</b> technique version is created for any new features</li>
                   <li>A new <b>major</b> version is created for any <b>architectural change</b> (such as refactoring)</li>
                 </ol>
-                <p>You can find your own Techniques written in the Technique Editor in the <b>User Techniques</b> category.</p>
+                <p>You can find your own techniques written in the technique editor in the <b>User Techniques</b> category.</p>
```

Comment:
"User Techniques" here is ok ?

Reply:
Don't look at the category in technique editor, you will not sleep

Reply:
yes because it's the name of the category